index.html

```diff
@@ -0,0 +1,285 @@
+  
+      <label for="country">Country</label>
+      <select id="country" name="country">
+        <option value="australia">Australia</option>
+        <option value="canada">Canada</option>
+        <option value="usa">USA</option>
+        <option value="india">India</option>
+        <option value="japan">Japan</option>
+        <option value="south korea">South Korea</option>
+      </select>
+  
+      <label for="subject">Subject</label>
+      <textarea id="subject" name="subject" placeholder="Write something.." style="height:200px"></textarea>
+  
+      <button type="submit" class= "button" value="Submit">Submit</button>
+    </form>
+  </div>
+        </section>
+    </main>
+    <footer>
+      <div class="footer-container">
+          <div class="footer-column">
+              <h3>Contact Us</h3>
+              <div class="social-icons">
+                  <a href="#" class="social-icon"><i class="fab fa-facebook"></i></a>
+                  <a href="#" class="social-icon"><i class="fab fa-twitter"></i></a>
+                  <a href="#" class="social-icon"><i class="fab fa-linkedin"></i></a>
+                  <a href="#" class="social-icon"><i class="fab fa-instagram"></i></a>
+              </div>
+          </div>
+          <div class="footer-column">
+              <h3>Discover</h3>
+              <ul>
+                  <li><a href="#services">Services</a></li>
+                  <li><a href="#pricing">Pricing</a></li>
+                  <li><a href="#portfolio">Portfolio</a></li>
+                  <li><a href="#contact">Contact</a></li>
+              </ul>
+          </div>
+          <div class="footer-column">
+              <h3>Get in Touch</h3>
+              <p>Email: support@digitalagency.com</p>
+              <p>Phone: +123 456 7890</p>
+              <p>Address: 123 Digital Ave, Tech City</p>
+          </div>
+      </div>
+      <div class="footer-bottom">
+          <p>Made with ❤️ ‍in India | &copy; 2024 Digital Agency | All rights reserved.</p>
+      </div>
+  </footer>
+  
+
+
+<!-- JavaScript to toggle the sidebar -->
+<script>
+    function showSidebar() {
+        document.querySelector('.sidebar').classList.add('active');
+    }
+
+    function hideSidebar() {
+        document.querySelector('.sidebar').classList.remove('active');
+    }
+      document.querySelector("form").addEventListener("submit", function (event) {
+          event.preventDefault();
+
+          const firstName = document.getElementById("fname").value.trim();
+          const email = document.getElementById("email").value.trim();
+          const country = document.getElementById("country").value;
+          const subject = document.getElementById("subject").value.trim();
+
+          let errors = [];
+
+          if (firstName === "") {
+              errors.push("First Name cannot be empty.");
+          }
+
+          const emailPattern = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+          if (!emailPattern.test(email)) {
```

Comment:
if (email === "") {
        errors.push("Email cannot be empty.");
    } else {
        const emailPattern = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
        if (!emailPattern.test(email)) {
            errors.push("Please enter a valid email address.");
        }
    }
    
to check the empty email add this for email validation

Reply:
updated